src/main/java/com/denizenscript/depenizen/bukkit/properties/auraskills/AuraSkillsPlayerProperties.java

```diff
@@ -0,0 +1,265 @@
+package com.denizenscript.depenizen.bukkit.properties.auraskills;
+
+import com.denizenscript.denizen.objects.PlayerTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import dev.aurelium.auraskills.api.AuraSkillsApi;
+import dev.aurelium.auraskills.api.skill.Skills;
+
+public class AuraSkillsPlayerProperties {
+
+    public static void register() {
+
+        // <--[tag]
+        // @attribute <PlayerTag.auraskills_level[<skill>]>
```

Comment:
similarly this tag could probably be the most useful if it just returned a MapTag of all the skills